book/02-using-atom/sections/06-customizing.asc

```diff
@@ -85,7 +85,7 @@ You can open this file in an editor from the _Atom > Open Your Config_ menu.
 ** `nonWordCharacters`: A string of non-word characters to define word boundaries
 ** `fontSize`: The editor font size
 ** `fontFamily`: The editor font family
-** `invisibles`: Specify characters that Atom renders for invisibles in this hash (use `false` to turn off individual character types)
+** `invisibles`: Specify characters that Atom renders as invisible in this hash (use `false` to turn off individual character types)
```

Comment:
You're right, @izuzak. I actually need to rewrite the definition of `invisibles` to better explain the key/value relationship in the hash, which is that the whitespace character (the key) will be replaced with the value.